doc/source/cd/methods/onlinefetdrift.ipynb

```diff
@@ -0,0 +1,175 @@
+{
```

Comment:
<p>"<code>n_bootstraps</code>  number" -&gt;  "<code>n_bootstraps</code>"</p><p>"the windows to be full" -&gt; "a window to be full" </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>